cumulusci/robotframework/tests/cumulusci/communities.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Resource  cumulusci/robotframework/CumulusCI.robot
+Library   Collections
+
+Suite Setup  run keywords
+     # Note: the first community name intentionally includes a unicode
+     # character to make sure we can handle it.
+...  Insure community exists       Kōkua   kokua
+...  AND  Insure community exists  Ohana   ohana
+
+*** Keywords ***
+Insure community exists
+    [Arguments]      ${community name}  ${url prefix}
+    [Documentation]  Creates a community with the given name if it doesn't exist
+
+    ${passed}=  run keyword and return status
+    ...  get community info  ${community name}
+
+    run keyword if  not $passed  run task  create_community
```

Comment:
`CreateCommunity` will throw `SalesforceException` if the target org doesn't have the Communities feature turned on - does that get caught as a test failure by Robot?

Reply:
yes, robot will see that as a test failure. This is an internal test, so we don't need to handle it since I've modified orgs/dev.json to turn the feature on.